### PR TITLE
Build warnings: contact us, corp info, supportedby

### PR DESCRIPTION
**Pull request by Garneauma**
Resolving build warnings part 7:
- Adding institutional information include file
- Adding organizational information include file
- Fixing reference to institutional info include file
- Fixing reference to organizational info include file
- Integrating "supported-by" include file content into files calling it

### CODE REVIEW

**Comment by Ricokola** [comment]
Pre-approved upon successful testing

File: templates/institutional/includes/inst-info.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+++++++++++++++++++++++++++++++++++
++ List of institutional links and head of institution image
++
++ Used by institutional pages
+++++++++++++++++++++++++++++++++++
+
+{%- endcomment -%}
+
+<section class="gc-crprt">
```
Comment (duboisp) [inline templates/institutional/includes/inst-info.html:11]: Would this code sample be in the "components" under "gc-crprt". It look very similar to the other code sample in organisation template.

I suggest to put both content sample under "components/gc-crprt/samples/" folder

File: templates/organizational/includes/org-info.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+++++++++++++++++++++++++++++++++++
++ List of organization links and head of organization image
++
++ Used by organizational pages
+++++++++++++++++++++++++++++++++++
+
+{%- endcomment -%}
+
+<section class="gc-crprt">
```
Comment (duboisp) [inline templates/organizational/includes/org-info.html:11]: Would this code sample be in the "components" under "gc-crprt"

File: templates/localnav/index-en.html [hunk 3]
```diff
@@ -48,7 +48,14 @@ <h3 class="h5"><a href="./task4/index-en.html">[Task 4]</a></h3>
 		</div>
 	</section>
 	<div class="pull-right col-xs-12 col-md-4">
-		{{>supported-by}}
+		<section class="lnkbx">
```
Comment (duboisp) [inline templates/localnav/index-en.html:51]: Comment: `lnkbx` CSS seem to be an utility component by itself

**Review by duboisp — CHANGES_REQUESTED** [review]
* [ ] Code sample should be in the "samples" folder instead of "includes"
* [ ] Move `gc-crprt` as a components outside templates

**Comment by Garneauma** [comment]
> * [x]  Code sample should be in the "samples" folder instead of "includes"
> * [x]  Move `gc-crprt` as a components outside templates

Done ! :)

**Review by duboisp — APPROVED** [review]

**Pull request merged**